web/webkit/SpecRunner.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Jasmine Spec Runner v2.4.0</title>
```

Comment:
I think it's fair to say that node is needed to run JS tests. Ideally we'll get to a place where we can make that no longer true, but until then that's fine IMO. Your call, ultimately.